app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,7 @@
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
     <link rel="stylesheet" href="https://use.typekit.net/mxd1nlu.css">
+    <script src="https://kit.fontawesome.com/e85e0c4e55.js" crossorigin="anonymous"></script>
```

Comment:
**issue:** adding scripts to `<head />` block the page rendering. We should either move it to right before `</body>` or use the [`async` attribute](https://javascript.info/script-async-defer)

Reply:
makes sense 😅